persist/sqlite/store.go

```diff
@@ -65,12 +67,23 @@ func (s *Store) transaction(fn func(*txn) error) error {
 	return fmt.Errorf("transaction failed (attempt %d): %w", attempt, err)
 }
 
+// Backup creates a backup of the database at the specified path. The backup is
+// created using the SQLite backup API, which is safe to use with a
+// live database.
+//
+// This function should be used if the database is already open in the current
+// process. If the database is not already open, use Backup.
+func (s *Store) Backup(ctx context.Context, destPath string) error {
+	return backupDB(ctx, s.db, destPath)
+}
+
 func sqliteFilepath(fp string) string {
 	params := []string{
 		fmt.Sprintf("_busy_timeout=%d", busyTimeout),
 		"_foreign_keys=true",
 		"_journal_mode=WAL",
 		"_secure_delete=false",
+		"_auto_vacuum=INCREMENTAL",
```

Comment:
I think this is only useful if you also use `incremental_vacuum` instead of the regular `VACUUM` which doesn't seem to be the case. 

> When the value of auto-vacuum is 2 or "incremental" then the additional information needed to do auto-vacuuming is stored in the database file but auto-vacuuming does not occur automatically at each commit as it does with auto_vacuum=full. In incremental mode, the separate [incremental_vacuum](https://sqlite.org/pragma.html#pragma_incremental_vacuum) pragma must be invoked to cause the auto-vacuum to occur.

[Source](https://sqlite.org/pragma.html#pragma_auto_vacuum)

Reply:
The plan is to eventually implement periodic incremental vacuums. Unfortunately, this needs to be set before you create the database or after a VACUUM.